### PR TITLE
hpre: fix coding style

### PR DESCRIPTION
**Pull request by barry19901226**
fix coding style

### CODE REVIEW

File: test/hisi_sec_test/test_hisi_sec.c [hunk 1]
```diff
@@ -1744,8 +1745,12 @@ int main(int argc, char *argv[])
 	if (!strcmp(argv[3], "-optype")) {
 		g_direction = strtoul((char*)argv[4], NULL, 10);
 		if (algtype_class == DIGEST_CLASS) {
```
Comment (hzhuang1) [inline test/hisi_sec_test/test_hisi_sec.c:1747]: Why does the mode number in the range of 0, 1 and 3? Do we need mode number 2?

File: test/hisi_sec_test/test_hisi_sec.c [hunk 2]
```diff
@@ -1744,8 +1745,12 @@ int main(int argc, char *argv[])
...
-			//0 is normal mode, 1 is HMAC mode.
+			//0 is normal mode, 1 is HMAC mode, 3 is long hash mode.
 			alg_op_type = g_direction;
+			if (g_direction == 3) {
+				alg_op_type = 0;
+				g_ivlen = 1;
+			}
```
Comment (hzhuang1) [inline test/hisi_sec_test/test_hisi_sec.c:1753]: It's not related to perf. Could you help moving it to another patch?

**Review by wangzhou — COMMENTED** [review]
fine to me

File: test/hisi_sec_test/test_hisi_sec.c (hunk 1 above)
Comment (wangzhou) [reply test/hisi_sec_test/test_hisi_sec.c:1747]: we should use getopt_long to do this

Comment (wangzhou) [reply test/hisi_sec_test/test_hisi_sec.c:1747]: maybe 2 will be used for other mode ：）
